src/EA.Weee.Web/Views/Shared/_CheckAnswersContentPartial.cshtml

```diff
@@ -163,12 +163,15 @@
                 </td>
             </tr>
         }
-        <tr class="govuk-table__row">
-            <th scope="col" class="govuk-table__header govuk-!-text-align-left" style="width: 28%;">Selling technique(s) that you use</th>
-            <td class="govuk-table__cell" colspan="2">
-                @Model.EeeData.SellingTechnique.ToSellingTechniqueType().GetDisplayName()
-            </td>
-        </tr>
+        @if (@Model.EeeData.SellingTechnique.IsDirectSelling || Model.EeeData.SellingTechnique.IsIndirectSelling)
+        {
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header govuk-!-text-align-left" style="width: 28%;">Selling technique(s) that you use</th>
```

Comment:
@will-hope98 Just thinking about this I wonder if we should always display the row it should just not display the value if it doesnt have value.

Reply:
@JohnPadgettCivica Would have to create a new partial I think. if you prefer to do it that way

Reply:
@will-hope98 Maybe move that into a partial and move the if statement to be inside the cell that renders the actual value so it displays nothing the row title but empty value when its not present.